apps/web/pages/reschedule/[uid].tsx

```diff
@@ -42,13 +42,17 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
   if (!booking) {
     return {
       notFound: true,
+    } as {
```

Comment:
Required to be used with `withEmbedSsr`. Without this `getServerSideProps` isn't passable to `withEmbedSsr`